packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -1261,3 +1215,19 @@ describe('IndexedDb: allowTabSynchronization', () => {
     });
   });
 });
+
+describe('IndexedDb', () => {
+  if (!IndexedDbPersistence.isAvailable()) {
+    console.warn('No IndexedDB. Skipping tests.');
+    return;
+  }
+
+  it('can re-open after close', async () => {
+    return withDb(2, async db => {
+      db.close();
+      await db.runTransaction('readwrite', V1_STORES, () =>
+        PersistencePromise.resolve()
+      );
```

Comment:
This test doesn't make an assertion, which is usually a sign that something's missing. If this would implicitly fail somehow it's worth adding a comment about that or perhaps adding a does not throw expectation.

Reply:
I am not sure how `expect().doesNotThrow()` interacts with Promises, so I added a comment instead.